src/index.js

```diff
@@ -12,6 +13,8 @@ const config = buildConfig();
 const server = initialize(config, app);
 const logger = loggers.get('default');
 
+childProcess.fork('./src/server/bulkDataProcessor.js');
+
```

Comment:
Do we need to add something here that will kill the child process upon exiting using ctrl+c? I messed around with some options but couldn't get anything to work, and never had issues without adding code here.

Reply:
nah